lib/tasks/data_hygiene.rake

```diff
@@ -32,4 +32,18 @@ namespace :data_hygiene do
       speech.save!(validate: false)
     end
   end
+
+  desc "Restore a deleted document by creating a new edition"
```

Comment:
I'd probably be a bit more explicit here, just so it's crystal clear for the developer who stumbles across this in a year's time.

```suggestion
  desc "Restore a deleted document by creating a new edition from its latest deleted draft"
```

Do you think the task should also only fail if the latest non-deleted edition is "superseded"?
Or should this task be generic enough to, e.g., retrieve a draft that someone has deleted but now wants to get back (but where the published document is otherwise in a healthy state)?

Reply:
Good shout on the description.

I was pondering the question about whether to constrain the task to documents where the latest non-deleted edition is "superseded". I couldn't really decide, but I couldn't see any obvious harm so I left it as is for simplicity's sake. If you'd feel more comfortable with ensuring it can only be used in the superseded case, I'd be happy to do so.